test/test_autograd.py

```diff
@@ -4601,6 +4601,18 @@ def test_checkpointing_without_reentrant(self, input_requires_grad):
             nn.Linear(nz_bottleneck, nz_inp)
         )
 
+        # Module holder for testing activation checkpointing with no_reentrant
+        # supports kwargs.
+        class MyModule(nn.Module):
+            def __init__(self, mod):
+                super().__init__()
+                self.module = mod
+
+            def forward(self, data):
+                return self.module(data)
+
+        module = MyModule(mod=module)
```

Comment:
Can you explain why we need `MyModule` as a module holder for the `nn.Sequential`?

Reply:
It allows us to pass in the kwarg on line 4726, if it was just the sequential, the name of the kwarg is not directly exposed